### DISCUSSION TITLE
Welcome to vs-code Discussions!

### DISCUSSION THREAD

**Discussion by michael-andreuzza** [Serendipity-Theme/vs-code#15] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by chaitanyabsprip** [comment]
First off, amazing theme!
I wanted to know which fonts you're using in the banner and the screenshots

**Reply by michael-andreuzza** [reply]
Hey, thank you!

Hey, thank you !

The banners is using Neue Machina and the font on the screenshot is using IBM Plex Mono
